.env.production

```diff
@@ -1 +1 @@
-VUE_APP_API_URL=
\ No newline at end of file
+VUE_APP_API_URL="http://antiplag.sk"
```

Comment:
Please do not change this value, create new variable. You simply cannot change the existing variables without breaking existing functionality. 
This is the variable for API url.